packages/teams-js/src/public/interfaces.ts

```diff
@@ -1262,3 +1262,41 @@ export interface ClipboardParams {
   /** Blob content in Base64 string format */
   content: string;
 }
+
+/**
+ * @hidden
+ * List of enter focus action items
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export enum EnterFocusActionItem {
+  /**
+   * Determines the direction to focus in app.
+   */
+  PreviousLandmark,
+  NextLandmark,
+  /**
+   * Determines if the focus should go to the particular content of the app.
+   */
+  Read,
+  Compose,
+}
+
+/**
+ * Return focus action items
+ */
+export enum ReturnFocusActionItem {
+  /**
+   * Determines the direction to focus in host for previous landmark.
+   */
+  PreviousLandmark,
+  /**
+   * Determines the direction to focus in host for next landmark.
+   */
+  NextLandmark,
+  /**
+   * Determines if the focus should go to the left rail of teams
+   */
+  GoToLeftRail,
```

Comment:
Is there a name for `GoToLeftRail` that you could use that doesn't include the word "left"? AppBar maybe?

I mainly ask because it always makes me shudder when we hardcode assumptions about *where* a piece of UI is into an enum like this. When the web design world inevitably decides that these bars have to go on the right or bottom or whatever in 5 years I'd like for this to still be accurate because we used Semantic naming instead of locational naming.

Reply:
Changed it to `GoToActivityFeed`